lib/core/i18n/i18n.utils.ts

```diff
@@ -44,3 +50,18 @@ export const getOrSetUserLocaleFromHeaders = async (req: IncomingMessage): Promi
 
   return preferredLocale;
 };
+
+export const getT = async (locale: string, ns: string) => {
+  const create = async () => {
+    const { _nextI18Next } = await serverSideTranslations(locale, [ns]);
+    const _i18n = i18next.createInstance();
+    _i18n.init({
+      lng: locale,
+      resources: _nextI18Next.initialI18nStore,
+      fallbackLng: _nextI18Next.userConfig?.i18n.defaultLocale,
+    });
+    return _i18n;
+  };
+  const _i18n = nexti18next != null ? nexti18next : await create();
+  return _i18n.getFixedT(locale, ns);
+};
```

Comment:
What are we doing here?

Reply:
It's a way to make use of the `t` function on server-side code (e.g. api, emails, etc.).

Reply:
Cool solution, but isn't there not an easier way to get this? 🤔

Reply:
Not sure there is @KATT, I've checked out a few blogs in this regard and it looks one of the "More complex" things to solve.